httpcore/_backends/sync.py

```diff
@@ -53,10 +170,15 @@ def start_tls(
         }
         with map_exceptions(exc_map):
             try:
-                self._sock.settimeout(timeout)
-                sock = ssl_context.wrap_socket(
-                    self._sock, server_hostname=server_hostname
-                )
+                if isinstance(self._sock, ssl.SSLSocket):
```

Comment:
We could do with a comment here explaining why/when we're using TLS and when we're using TLS-in-TLS